version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.8.2-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.8.3-SNAPSHOT"
```

Comment:
This file shouldn't be edited manually, it's being modified automatically when releasing the new version.